source/cloud-security/gcp/posture-management.rst

```diff
@@ -217,7 +217,7 @@ Identity and access management anomalous activity
 Posture management result
 -------------------------
 
-Visualize the Google Cloud posture management results by navigating to **Modules** > **Security events**. Filter for the rule IDs ``100200`` and ``100201``.
+Visualize the Google Cloud posture management results by navigating to **Threat hunting**. Filter for the rule IDs ``100200`` and ``100201``.
```

Comment:
**nitpick**: The application is called as `Threat Hunting`.

Reply:
Done